modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/data/provider/ContentDashboardDataProvider.java

```diff
@@ -323,11 +344,22 @@ private TermsAggregation _getTermsAggregation(
 			new IncludeExcludeClauseImpl(
 				assetCategoryIds.toArray(new String[0]), new String[0]));
 
+		termsAggregation.setMinDocCount(minDocCount);
+
 		return termsAggregation;
 	}
 
+	private TermsAggregation _getTermsAggregation(
+		AssetVocabulary assetVocabulary, Set<String> assetCategoryIds,
+		String termsAggregationName) {
+
+		return _getTermsAggregation(
+			assetVocabulary, assetCategoryIds, null, termsAggregationName);
```

Comment:
Not a big fan of null values that are hard to read :(.

Reply:
I know, me neither, but the Liferay Search framework forced me to do that, let me see, how i can improve it

Reply:
Done. Thanks!